### DISCUSSION TITLE
How to install OpenBB SDK?

### DISCUSSION THREAD

**Discussion by hminle** [OpenBB-finance/OpenBB#3495] [Q&A] [answered]
Hi guys,

How could we install OpenBB SDK via `pip`?

Thanks

**Comment by northern-64bit** [comment, marked as answer]
Hello @hminle! There is a guide that explains how to install the SDK properly with pip: https://docs.openbb.co/sdk/quickstart/installation
I hope that this helps!

**Reply by hminle** [reply]
Thank @northern-64bit , I got it.

**Reply by dMLTquant** [reply]
@hminle if you want to try out OpenBB SDK without having to worry about install have a look at this (you only need a GitHub and a free GitPod account): [GitHub > dMLTquant > openbb_sdk_exporation](https://github.com/dMLTquant/openbb_sdk_exporation)

**Reply by hminle** [reply]
@dMLTquant thank you so much, I am looking at your repos. It's interesting.
